wpeverest/everest-forms
# Module - Style Customizer

### All Submissions:


### Changes proposed in this Pull Request:
Previously. Style Customizer is addon and moved to module.

### How to test the changes in this Pull Request:
Check the entire functionality and verify if the existing settings are working when switching to the module.

### Types of changes:




### Other information:




### Changelog entry

> Module - Style Customizer.
